_posts/2020-03-01-laniakea-release.md

```diff
@@ -0,0 +1,23 @@
+---
+site: Italy
```

Comment:
@mtangaro I guess we can only merge this when this site exists. Should we wait?

Othwerwise, this looks great.